### ISSUE TITLE
Obsolete GO:0062198  rDNA replication pause site heterochromatin ? 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#19178]
Hello, 

GO:0062198  rDNA replication pause site heterochromatin has no annotations. It's defined as 'A region of heterochromatin associated with sites in rDNA repeats where replication forks pause.' - but I dont think this is a specific cellular component. 

There are no annotations. Paper cited to create the term is PMID: 25417108

@colinlog @RLovering @ValWood 

Do you agree with obsoletion ? 

Thanks, Pascale

**Comment by ValWood** [comment]
I do, we have not used it for the supporting publication. Sounds more like a SO term

**Comment by colinlog** [comment]
I would like to email Rob Martienssen and ask where he would house this
term in the GO or SO. As far as I can see, you are correct Pascale, it is
not clear that this is a cell component. It is more like an intermediate in
replication. Is there a 'replication fork collision with RNA polymerase'
term? Jesper Svejstrup is an author who has done quite some work on this.
Basically the process would be a transcription and replication term: how
cells avoid interference between these two processive DNA tracking engines
that are the RNA and DNA polymerases.  I can CC Jesper Svejstrup in my
email to Rob Martienssen. Hoep t get to this today...

Pascale how do I find authors whose papers have been used in GO annotation?
In other words; how can I find GO terms that are associated with a
particular author?

**Comment by pgaudet** [comment]
> Pascale how do I find authors whose papers have been used in GO annotation?

Well in this case, as I mentioned in the first comment, there are no annotations. 
Otherwise we look at annotations in AmiGO or QuickGO (I can show you).

Thanks, Pascale

**Comment by ValWood** [comment]
This term isn't used by us for GO anntoation. It seems to have been created  for FYPO logical def. If this is the case the logical def can use SO instead.
https://www.pombase.org/reference/PMID:25417108

@mah11

**Comment by mah11** [comment]
FYPO does not use GO:0062198.

**Comment by ValWood** [comment]
OK in that case I don't know where this term comes from.
can be obsoleted.

**Comment by ValWood** [comment]
Oh here is the history:
https://github.com/geneontology/go-ontology/issues/18249

**Comment by mah11** [comment]
What I said in #18249 still holds - if GO:0062198 is kept, FYPO can change an existing logical definition to use it, but if the GO term goes obsolete FYPO can do without the logical def. (FYPO has not yet used GO:0062198.)

**Comment by pgaudet** [comment]
But @mah11 You write above that 'FYPO does not use GO:0062198.'  - does it ?

**Comment by mah11** [comment]
Not now -- "does not" is *present* tense.

As my second comment indicates, FYPO could use GO:0062189 in the near *future* if it is retained.

I'd like to see GO second-guess its decisions a bit less, since the issue seemed sorted following discussion for #18249 -- @ukemi even said it was considered on an editors' call -- but it remains the case that FYPO will adjust to whatever the outcome of this ticket is.

**Comment by ValWood** [comment]
Anyway @colinlog  no need to contact Rob, but he will be pleased about these changes because he is one of the people who said we should remove the 'chromatin silencing' in favour of heterochromatin assembly. I will ask him to look over these changes once completed.

**Comment by pgaudet** [comment]
HI @mah11 Sorry about the flip flopping. We're trying to take a more global approach to make the guidelines for creating terms or not more consistent.

**Comment by pgaudet** [comment]
Dear all, 

The proposal has been made to obsolete GO:0062198 rDNA replication pause site heterochromatin. The reason for obsoletion is that this is not a specific cellular component, it is a type of sequence, which should be captured with a Sequence Ontology term. 

There are no annotations, no mappings to this term. This term is not present in any subset. 

You can comment in the ticket:
https://github.com/geneontology/go-ontology/issues/19178

We are opening a comment period for this proposed obsoletion. We’d like to proceed with the obsoletion this term on April 8th, 2020. Unless objections are received by April 8th, 2020, we will assume that you agree to this change.

Thanks, Pascale
